website/source/docs/providers/aws/r/alb_target_group.html.markdown

```diff
@@ -31,7 +31,8 @@ resource "aws_vpc" "main" {
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the target group.
+* `name` - (Optional, Forces new resource) The name of the target group. If omitted, Terraform will assign a random, unique name.
+* `name_prefix` - (Optional, Forces new resource) Creates a unique name beginning with the specified prefix. Conflicts with `name`.
```

Comment:
Could you please include the character limits here in the documentation? Also, why the `6` character limit for the prefix?